frontend/src/components/personal/PasswordChangeForm.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useCallback } from 'react'
+import {useDispatch} from 'react-redux'
+import Collapsible from 'react-collapsible'
+import { CardActions, Button, TextField} from '@material-ui/core'
+
+import {updateUserPassword,
+    userPreferencesRequestFailed, userPreferencesRequestInitiated } from '../../actions/prefAndProfileActions'
+  import {useStyles} from './ProfilePreferencesPage.styles'
+
+
+export default function PasswordChangeForm (): React.ReactElement {
+    const [oldPassword, setOldPassword] = useState('')
+    const [newPassword, setNewPassword] = useState('')
+    const [confirmPassword, setConfirmPassword] = useState('')
+
+    const dispatch = useDispatch()
+  
+      const onChangeOldPassword = useCallback((e) => {
+        setOldPassword(e.target.value)
+      }, [setOldPassword])
+  
+      const onChangeNewPassword = useCallback((e) => {
+        setNewPassword(e.target.value)
+      }, [setNewPassword])
+  
+      const onChangeConfirmPassword = useCallback((e) => {
+        setConfirmPassword(e.target.value)
+      }, [setConfirmPassword])
+  
+    const savePasswordChange = useCallback((e) => {
```

Comment:
I think it can be made as a custom hook.

Reply:
it can be inside `submit` function of Formik form manager - try catch between dispatch success or dispatch error